test/common/phantom-command

```diff
@@ -1,4 +1,4 @@
 #!/bin/sh
 BASE=$(dirname $0)
 PATH="$BASE/../../node_modules/.bin:$PATH"
-exec phantomjs --ignore-ssl-errors=true --ssl-protocol=any $@
+exec phantomjs --ignore-ssl-errors=true --ssl-protocol=any --web-security=false $@
```

Comment:
Why is --web-security=false necessary?

Reply:
pkg/ovirt connects to oVirt REST API served from other domain than the cockpit's host. PhantomJS forbids that by default.